### PR TITLE
Reorder non-basemap layers at vizjson

### LINKED ISSUE #4374: reorder label layers at vizjson
Upon generation of the viz.json, not always are layers ordered by its order. This has no problem regarding rendering but should be consistent and keep that order to ease reading (for debugging etc.)

/ cc @alonsogarciapablo

### PR DESCRIPTION
**Pull request by Kartones**
Closes #4374

### CODE REVIEW

**Comment by Kartones** [comment]
@alonsogarciapablo  CR please (while I see if I must stabilize any test), tested locally and works fine, will test at staging inmediately

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4980/))

**Comment by alonsogarciapablo** [comment]
:+1:

File: app/models/visualization/vizjson.rb [hunk 1]
```diff
@@ -107,12 +107,15 @@ def layers_for(visualization)
...
-          layers_data.push( named_maps_presenter.to_poro )
+          layers_data.push(named_maps_presenter.to_poro)
         else
           named_maps_presenter = nil
-          layers_data.push( layer_group_for(visualization) )
+          layers_data.push(layer_group_for(visualization))
         end
-        layers_data.push( other_layers_for( visualization, named_maps_presenter ) )
+        layers_data.push(other_layers_for(visualization, named_maps_presenter))
+
```
Comment (alonsogarciapablo) [inline app/models/visualization/vizjson.rb]: I'd remove these whitespaces.

**Comment by Kartones** [comment]
works like a charm

**Comment by Kartones** [comment]
@alonsogarciapablo changed a bit the code to support nils, as tests not always have all "expected" layers :trollface:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4983/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/4991/))

**Comment by alonsogarciapablo** [comment]
:+1:

**Pull request merged**